lib/config.js

```diff
@@ -149,6 +149,10 @@ var normalizeConfig = function (config, configFilePath) {
     throw new Error('Invalid configuration: client.args must be an array of strings')
   }
 
+  if ('browsers' in config && Array.isArray(config.browsers) === false) {
```

Comment:
Please use `if (config.browsers &&` instead, to be consistent with the rest of the code here.